flow360/component/flow360_params/flow360_params.py

```diff
@@ -943,8 +944,22 @@ def check_bet_disks_3d_coefficients_in_polars(cls, values):
         return _check_bet_disks_3d_coefficients_in_polars(values)
 
 
-class PorousMediumVolumeZone(Flow360BaseModel):
-    """:class:`PorousMediumVolumeZone` class"""
+# pylint: disable=too-few-public-methods
+class PorousMedium(PorousMediumVolumeZone):
```

Comment:
Why does PorousMedium extend PorousMediumVolumeZone?

Should we have a base abstract class instead that is inherited by both the above classes?